misk-launchdarkly-core/src/test/kotlin/misk/feature/launchdarkly/LaunchDarklyFeatureFlagsTest.kt

```diff
@@ -29,80 +29,101 @@ import org.mockito.Mockito.verify
 internal class LaunchDarklyFeatureFlagsTest {
   private val client = mock(LDClientInterface::class.java)
   private val moshi = Moshi.Builder()
-      .add(KotlinJsonAdapterFactory()) // Added last for lowest precedence.
-      .build()
+    .add(KotlinJsonAdapterFactory()) // Added last for lowest precedence.
+    .build()
   private val featureFlags: FeatureFlags = LaunchDarklyFeatureFlags(client, moshi)
 
   @BeforeEach
   fun beforeEach() {
-    Mockito.`when`(client.initialized()).thenReturn(true)
+    Mockito.`when`(client.isInitialized).thenReturn(true)
   }
 
   @Test
   fun getEnum() {
     Mockito
-        .`when`(client.stringVariationDetail(anyString(), any(LDUser::class.java), anyString()))
-        .thenReturn(EvaluationDetail(EvaluationReason.targetMatch(), 1, "TYRANNOSAURUS"))
+      .`when`(client.stringVariationDetail(anyString(), any(LDUser::class.java), anyString()))
+      .thenReturn(
+        EvaluationDetail.fromValue(
+          "TYRANNOSAURUS", 1, EvaluationReason.targetMatch()
+        )
+      )
 
     val feature = featureFlags.getEnum<Dinosaur>(
-        Feature("which-dinosaur"), "abcd",
-        Attributes(mapOf(
-            "continent" to "europa",
-            "platform" to "lava"
-        ), mapOf(
-            "age" to 100000
-        )))
-
-    assertThat(feature).isEqualTo(
-        Dinosaur.TYRANNOSAURUS)
+      Feature("which-dinosaur"), "abcd",
+      Attributes(
+        mapOf(
+          "continent" to "europa",
+          "platform" to "lava"
+        ),
+        mapOf(
+          "age" to 100000
+        )
+      )
+    )
+
+    assertThat(feature).isEqualTo(Dinosaur.TYRANNOSAURUS)
 
     val userCaptor = ArgumentCaptor.forClass(LDUser::class.java)
     verify(client, times(1))
-        .stringVariationDetail(eq("which-dinosaur"), userCaptor.capture(), eq(""))
+      .stringVariationDetail(eq("which-dinosaur"), userCaptor.capture(), eq(""))
 
     val user = userCaptor.value
 
     // User fields are package-private, so we fetch it with reflection magicks!
     val customField = LDUser::class.java.getDeclaredField("custom")
     customField.isAccessible = true
     @Suppress("unchecked_cast")
-    val customAttrs = customField.get(user) as Map<String, LDValue>
+    val customAttrs = customField.get(user) as Map<UserAttribute, LDValue>
 
     val privateAttrsField = LDUser::class.java.getDeclaredField("privateAttributeNames")
     privateAttrsField.isAccessible = true
     @Suppress("unchecked_cast")
     val privateAttrs = privateAttrsField.get(user) as Set<String>
-
-    assertThat(customAttrs.getValue("continent").stringValue()).isEqualTo("europa")
-    assertThat(customAttrs.getValue("platform").stringValue()).isEqualTo("lava")
-    assertThat(customAttrs.getValue("age").intValue()).isEqualTo(100000)
-    assertThat(privateAttrs).isEqualTo(setOf("continent", "platform", "age"))
+    val continent = UserAttribute.forName("continent")
```

Comment:
Keys must now be wrapped in [UserAttribute](http://launchdarkly.github.io/java-server-sdk/com/launchdarkly/sdk/UserAttribute.html)